test/components/views/rooms/__snapshots__/PinnedEventTile-test.tsx.snap

```diff
@@ -51,7 +51,7 @@ exports[`<PinnedEventTile /> should render pinned event 1`] = `
       <span
         class="mx_MessageTimestamp mx_PinnedEventTile_timestamp"
       >
-        Thu, Jan 1 1970 00:00:00
+        Thu, Jan 1, 1970, 24:00
```

Comment:
24:00? seriously?

(also, why has this lost its seconds component?)

Reply:
From the other formats the magical do-everything function supported seconds felt out of place, though that ideally would be done in a separate PR if you call for such. Seconds makes little sense when you're referring to an entirely different year

Reply:
`formatDate` final case gained a `showSeconds=false`